### PR TITLE
Add PropertiesConfig integration test

### PR DESCRIPTION
**Pull request by mprimi**

### CODE REVIEW

**Comment by mprimi** [comment]
@tgianos Small change ready for review

**Comment by mprimi** [comment]
How did I manage to *reduce* coverage by adding a test??

**Comment by tgianos** [comment]
Cause coveralls is weird with rounding errors it seems.

File: genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+/**
+ * Integration test for PropertiesConfig values binding.
+ *
+ * @author mprimi
+ * @since 3.1.0
+ */
+@Category(IntegrationTest.class)
+@RunWith(SpringRunner.class)
+@TestPropertySource(locations = "classpath:/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.properties")
```
Comment (tgianos) [inline genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java]: You could likely also accomplish this (with a yml file as well) by just putting a properties file in source/test/resources and calling it like `application-propstest.yml` and then activating the `@Profile("propstest")` for this test. That should load that properties source. Also I wouldn't put this properties file explicitely in this location as it inconsistent with the rest of our locations.

File: genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+@SpringBootTest()
```
Comment (tgianos) [inline genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java]: Do you need to set anything here to make sure a full app server is not loaded?

File: genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java (hunk 1 above)
Comment (mprimi) [inline genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java]: I like the idea of profile, it's more similar to how properties are normally loaded (and it's YAML). I'll switch to this and move this file with the other profiles.

File: genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java (hunk 2 above)
Comment (mprimi) [inline genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java]: Without args defaults to a mock server, but i could make it explicit.

File: genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java (hunk 1 above)
Comment (mprimi) [inline genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java]: Scratch that. For some reason, profile does not work. I tried to figure it out for an hour but in the interest of time, I'm leaving this as is (but moving it to the top level test resource directory)

**Comment by mprimi** [comment]
@tgianos moved the file, but kept everything else the same (as i couldn't get loading via profile to work).

**Pull request merged**